andela/ah-shakas
# #160577492 Enable users to see time it takes to read an article

**What does this PR do?**
- It implements functionality for displaying time taken to read an article

**Description of task to be completed**
- Users should be able to see time it takes to read an article

**How should this be manually tested?**
Clone the GitHub repository: git clone https://github.com/andela/ah-shakas on your machine

Navigate to the project folder: cd ah-shakas

Checkout out to this PR's branch: git checkout ft-time-to-read-article-160577492

Create a virtual environment using pipenv then activate using the command pipenv shell

Install dependancies: pipenv install

To run tests, run the command python manage.py test

To run the app on Postman, run the command python manage.py runserver

- Send a `GET`/ `POST` request to the endpoint `/api/articles/` with necessary data for a `POST` request as below:
```
{
    "article":{
        "title": "my title",
        "description": "my description",
        "body": "this is a very nice article. I love it"
    }
}
```
- Notice the returned  article json has a field for `time_to_read`

**Screenshot**
<img width="1063" alt="screen shot 2018-10-09 at 21 11 11" src="https://user-images.githubusercontent.com/34505753/46689210-fca04780-cc07-11e8-957c-bb933c19e757.png">

**Relevant Pivotal Tracker story**
[160577492](https://www.pivotaltracker.com/n/projects/2198533/stories/160577492)
